### DISCUSSION TITLE
Error Occurred: [OOMKilled] job failed because of Insufficient memory

### DISCUSSION THREAD

**Discussion by dataZhi** [PaddlePaddle/PaddleSpatial#154] [BaiduKDDCup2022] [answered]
Dear Organizer,
I got an error of "Error Occurred": "[OOMKilled] job failed because of Insufficient memory", my team name is 'meatTeam'. Submit Time is 2022-06-07 09:02.
And  I'd like to get the detail of the error message.

**Comment by ramber1836** [comment, marked as answer]
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/19561583/172374634-6fdde4a0-9536-4e89-91d0-c328bac8823e.png">


No detailed log is available when OOMKilled is encountered.
